dockerfiles/theia/src/extensions.json

```diff
@@ -26,6 +33,20 @@
             "source": "https://github.com/eclipse/che-theia-hosted-plugin-manager-extension.git",
             "checkoutTo": "latest-deps",
             "type": "git"
+        },
+        {
+            "name": "@eclipse-che/theia-task-extension",
+            "source": "https://github.com/eclipse/che-theia-task-plugin",
```

Comment:
Task plugin(actually for now extension) use terminal server side, without this part it's compiles but doesn't work.

Reply:
removed